src/zac/demo/mor/tests/factories.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from factory.django import DjangoModelFactory as Dmf
+from zac.accounts.models import User
+from ...models import SiteConfiguration
+
+
+class UserFactory(Dmf):
+
+    class Meta:
+        model = User
+
+    username = 'test'
+    password = 'test1234'
+
+
+class SiteConfigurationFactory(Dmf):
+    class Meta:
+        model = SiteConfiguration
+
+    ztc_catalogus_uuid = 'f7afd156-c8f5-4666-b8b5-28a4a9b5dfc7'
```

Comment:
probably should be a `factory.Faker('uuid')` or something like that

Reply:
Not sure, I have to ask to to Alex that wrote the tests is this string is fixed or not

Reply:
still, if it is, it should be specified in the test itself and not in the factory. A factory is meant to generate (random) data conforming to a pattern, it is _not_ a replacement for fixtures. Tests should not break if you change details in factories.